website/content/docs/k8s/helm.mdx

```diff
@@ -378,6 +397,17 @@ Use these links to navigate to a particular top-level stanza.
 
       - `secretKey` ((#v-global-acls-replicationtoken-secretkey)) (`string: null`) - The key of the Kubernetes or Vault secret.
 
+    - `partitionToken` ((#v-global-acls-partitiontoken)) - partitionToken references a Vault secret containing the ACL token to be used in non-default partitions.
+      This value should only be provided in the default partition and only when setting
+      `global.secretsBackend.vault.enabled` to true.
+      We will use the value of the secret stored in Vault to create an ACL token in Consul with the value of the
+      secret as the secretID for the token.
+      In non-default, partitions set this secret as the `bootstrapToken`.
```

Comment:
"We" is only used where we are making recommendations. Otherwise we use "you" when referring to the user. Is this something that the user will do or that Consul will do on behalf of the user? If the latter, consider using "Consul will ..."